pkgs/applications/misc/autospotting/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "AutoSpotting-${version}";
+  version = "unstable-acbadb09a0ab2c2d3f5dcc984594f65d7904d587";
```

Comment:
use a date instead of a hash as version identifier. See the naming guidelines in the Nixpkgs manual.

Reply:
Fixed. Much cleaner too glad to know that now.